.github/workflows/test.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
Could you add 3.10-dev also here? In sphinx-doc/sphinx, it has been installed via deadsnakes/action.
https://github.com/sphinx-doc/sphinx/blob/3.x/.github/workflows/main.yml#L30-L33
https://github.com/sphinx-doc/sphinx/blob/3.x/.github/workflows/main.yml#L44-L48

Reply:
I added py3.10-dev to the target versions list.
It seems to work fine ;-)